app/src/main/res/values/dimens.xml

```diff
@@ -176,9 +176,9 @@
   <dimen name="previous_button_exploration_view_padding_start">24dp</dimen>
   <dimen name="previous_button_question_view_padding_start">24dp</dimen>
   <dimen name="previous_responses_item_exploration_split_view_margin_top">24dp</dimen>
-  <dimen name="previous_responses_item_exploration_view_margin_top">24dp</dimen>
+  <dimen name="previous_responses_item_exploration_view_margin_top">0dp</dimen>
   <dimen name="previous_responses_item_question_split_view_margin_top">24dp</dimen>
-  <dimen name="previous_responses_item_question_view_margin_top">24dp</dimen>
+  <dimen name="previous_responses_item_question_view_margin_top">0dp</dimen>
```

Comment:
If there is a padding / margin of `0dp` remove that and use `space_0dp` where it was used in code.

Reply:
Also merge with latest develop

Reply:
@rt4914 Ok, and should I let this dimension be 0dp now or revert it back to 24dp?

Reply:
@rt4914 so are we following that `space_0dp` suggestion which we concluded while renaming all the dimensions during SLOP

Reply:
> @rt4914 Ok, and should I let this dimension be 0dp now or revert it back to 24dp?

You just have to use space_0dp in place of a named dimension with a value of 0dp. So don't revert back.

Reply:
Basically all dimens values should be of some value other than 0dp and whereever you want to use `0dp` use `space_0dp`.

Reply:
This does not seem to be solved yet.